standards/cms/src/pkcs7_compat.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{
```

Comment:
Could we add some top level documentation about this module? Just a short description about the incompatibility